docs/api/projects.rst

```diff
@@ -21,11 +21,6 @@
 .. automodule:: readthedocs.projects.models
     :members:
 
-:mod:`readthedocs.projects.search_indexes`
```

Comment:
This was removed in a cleanup (https://github.com/rtfd/readthedocs.org/pull/4039/)